server/jms/src/main/java/org/hawkular/apm/server/jms/RetryCapableMDB.java

```diff
@@ -95,8 +98,7 @@ public void onMessage(Message message) {
             process(tenantId, items, retryCount);
 
         } catch (Exception e) {
-            // TODO: Handle nak of JMS message?
```

Comment:
Here we're at quite a high level but I still think it'd be good not to retry on every kind of failure.